DTM/CherrylandMC/map.json

```diff
@@ -146,6 +146,10 @@
 			"type": "build", "evaluate": "deny", "teams": ["red", "blue"],
 			"regions": ["chests"],
 			"message": "&cYou may not physically edit the TNT chests."
+		},
+		{
+			"type": "block-explode", "evaluate": "deny", "teams": ["blue", "red"],
+			"regions": ["blue-spawn-protection", "red-spawn-protection"],
```

Comment:
The comma after the regions here is unnecessary/incorrect, else the map won't load because it will look for the next attribute/flag/declaration/filter or whatever else follows. With it being inexistent, the JSON is invalidated.